### PR TITLE
 RCPS-144: stud-info-mgt-app: Even if you don't enter any information…

### PR DESCRIPTION
**Pull request by jshrq**
…, anyone can log in.

### CODE REVIEW

File: src/main/java/com/student/user/app/controllers/item/AddStudentController.java [hunk 1]
```diff
@@ -61,7 +62,7 @@ public class AddStudentController implements Initializable {
     @FXML
     private Button studentAddButton;
 
-    private StudentFacade studentFacade = new StudentFacadeImpl();
+    private StudentFacade studentFacade = new StudentFacadeImpl(new StudentDaoImpl());
```
Comment (RoCS2024) [inline src/main/java/com/student/user/app/controllers/item/AddStudentController.java]: Daos should not be visible in UI

File: src/main/java/com/student/user/app/controllers/item/UpdateStudentController.java [hunk 2]
```diff
@@ -58,7 +59,7 @@ public class UpdateStudentController {
     @FXML
     private Button studentUpdateButton;
 
-    private StudentFacade studentFacade = new StudentFacadeImpl();
+    private StudentFacade studentFacade = new StudentFacadeImpl(new StudentDaoImpl());
```
Comment (RoCS2024) [inline src/main/java/com/student/user/app/controllers/item/UpdateStudentController.java]: Daos should not be visible in UI

File: src/main/java/com/student/user/app/controllers/main/MainController.java [hunk 3]
```diff
@@ -50,13 +108,13 @@ void changeVisibility(ActionEvent event){
...
     }
 
-    public void logButtonOnAction(ActionEvent event) throws IOException {
-        Parent root = FXMLLoader.load(getClass().getResource("/views/UserList.fxml"));
-        stage = (Stage)((Node)event.getSource()).getScene().getWindow();
-        scene = new Scene(root);
-        stage.setScene(scene);
-        stage.show();
-    }
+//    public void logButtonOnAction(ActionEvent event) throws IOException {
```
Comment (RoCS2024) [inline src/main/java/com/student/user/app/controllers/main/MainController.java]: why comment?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Comment by jshrq** [comment]
> see comments

all done sir!

**Review by zhairanicolepacheco — APPROVED** [review]

**Review by RoCS2024 — APPROVED** [review]

**Pull request merged**